src/AzureAuth/Ado/TokenFormatting.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Authentication.AzureAuth.Ado
+{
+    using System;
+    using System.Text;
+
+    /// <summary>
+    /// <see cref="string"/> extension methods for formatting tokens.
+    /// </summary>
+    public static class TokenFormatting

```

Comment:
```suggestion
    public static class TokenFormatting
    public static class TokenFormattingExtensions
```

For extionsion class, I would like to add `extensions` as a suffix.